src/extension.ts

```diff
@@ -58,8 +62,18 @@ export function activate(context: ExtensionContext) {
 		window.showInformationMessage('Disabled Discord Rich Presence for this workspace.');
 	});
 
+	// Register the `discord.reconnect` command
+	const reconnecter = commands.registerCommand('discord.reconnect', async () => {
+		if (rpc) try { await destroyRPC(); } catch {}
+		initRPC(config.get('clientID'), true);
+		window.showInformationMessage('Reconnecting to Discord RPC');
+
+		if (statusBarIcon)
```

Comment:
Not liking how you newline the if-statement here. This is def. a style change and should not happen, see 5 lines above how you inline it usually when its one statement.